requirements.txt

```diff
@@ -1,2 +1,17 @@
-    python-docx==0.8.10
-    python-pptx==0.6.18
+astroid==2.4.2
```

Comment:
Just put those libraries here which are being used by this project. `pip freeze > requirements.txt` doesn't work well in all scenarios (inter-dependency resolution).

Reply:
Commited changes in requirements, kindly review.